modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-pools-hcp_{context}"]
+= Machine pools in {hcp-title} clusters
+
+In {hcp-title} clusters, the {hcp} spans across three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can only have one machine pool. 
+
+Additionally, machine pools in {hcp-title} clusters each upgrade independently and the versions of the machine pools must remain within two minor (Y-stream) versions of the control plane. While machine pools created with your cluster use the same version, since machine pools upgrade independently, the machine pools must remain within 2 y-stream versions of the hsted control plane.
```

Comment:
```suggestion
Each machine pool in an {hcp-title} cluster upgrades independently. Because the machine pools upgrade independently, they must remain within 2 minor (Y-stream) versions of the hosted control plane.
```
It might behoove you to add a more practical example as well, such as: "For example, if your hosted control plane is 4.16.z, your machine pools must be at least 4.14.z" (or something along those lines).